### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by rhaschke**
Various small improvements. See individual commit logs for details. @rickstaa, please have a look.
Most major change: Renaming `demo_gazebo.launch` -> `franka_gazebo.launch`, because I plan to introduce a `demo_gazebo.launch` based on standard ROS packages only, i.e. without `franka_ros/franka_gazebo`.

### CODE REVIEW

File: launch/franka_gazebo.launch [hunk 1]
```diff
@@ -8,8 +8,7 @@
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
 
-  <!-- By default, we won't load or override the robot_description -->
-  <arg name="load_robot_description" default="false"/>
```
Comment (rhaschke) [inline launch/franka_gazebo.launch]: We cannot offer that as an argument, because the Gazebo-specific URDF needs to be generated and loaded.

**Comment by rickstaa** [comment]
@rhaschke I agree with all your changes. The code is a lot cleaner now, thanks! All my manual tests seem to pass.

**Review by rickstaa — APPROVED** [review]

File: launch/franka_gazebo.launch (hunk 1 above)
Comment (rickstaa) [reply launch/franka_gazebo.launch]: Good catch. This also needs to be applied to the `melodic-branch` simulation: 

https://github.com/ros-planning/panda_moveit_config/blob/aebdd0afaf9c8fcb623802225b0c989caac206d0/launch/demo_gazebo.launch#L12). 

I will add a comment to #68.

Comment (rhaschke) [reply launch/franka_gazebo.launch]: My idea is to merge the Noetic into the Melodic branch, once we have settled everything.
The MoveIt configs should work on both distros in the same fashion and franka_ros has the same versions for Melodic and Noetic as well, right?

**Pull request merged**